org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/util/JavaProjectClasspathChangeAnalyzer.java

```diff
@@ -110,13 +173,6 @@ public boolean isRelevantPackageFragmentRootChange(IJavaElementDelta delta) {
 				);
 	}
 
-	private boolean isAttachmentChangeFlagOnly(IJavaElementDelta child) {
```

Comment:
I'm aware of client code that uses reflection to access this method. Can we keep the name, add a comment and make it protected instead?

Reply:
And adjust the implementation accordingly of course